Objective-C/Tests/LogTestOld.m

```diff
@@ -0,0 +1,450 @@
+//
+//  LogTest.m
+//  CouchbaseLite
+//
+//  Copyright (c) 2018 Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "CBLTestCase.h"
+#import "CBLLog+Logging.h"
+#import "CustomLoggerOld.h"
+
+@interface FileLoggerBackup: NSObject
+
+@property (nonatomic, nullable) CBLLogFileConfiguration* config;
+
+@property (nonatomic) CBLLogLevel level;
+
+@end
+
+@interface LogTestOld : CBLTestCase
+
+@end
+
+@implementation LogTestOld {
+    FileLoggerBackup* _backup;
+    CBLLogLevel _backupConsoleLevel;
+    CBLLogDomain _backupConsoleDomain;
```

Comment:
I think, there is no need to backup anything for testing the old API. The backup is for the case that we enable a console log in the base test case for debugging purpose.

Reply:
How do you handle the new and the old API are both used as the new logging tests and the old logging tests will be both run? Doesn't an exception will be thrown?

Reply:
> How do you handle the new and the old API are both used as the new logging tests and the old logging tests will be both run? Doesn't an exception will be thrown?

Both suites run one after the other. I've used `isInitialized` to allow this only once, for the old custom logger needed in the old tests. Without that check, it will throw indeed